.github/workflows/build.yml

```diff
@@ -138,6 +138,8 @@ jobs:
           retention-days: 7
 
   build-docker:
+    permissions:
+      id-token: write
```

Comment:
Isn't this a bit much?

Reply:
Based on the Action's documentation, it isn't.